addons/portal_rating/static/src/js/portal_composer.js

```diff
@@ -74,6 +74,24 @@ PortalComposer.include({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * Only returns a Promise (to prevent multiple submissions) that is 
+     * never resolved when form is valid, otherwise user cannot re-send
+     * after correcting invalid fields.
+     *
+     * @override
+     * @private
+     * @returns {Promise}
+     */
+    _onSubmitButtonClick: function () {
+        var $form = this.$('textarea');
```

Comment:
Done and done. As far as I have seen, only 'website_sale' will be the only other module affected and it currently also does nothing without notifying users when submitting reviews with no message.

Reply:
Both renaming and separate commit done. Even though I updated my changes to only affect 'portal_rating', since the solution is still not super great, it might still want to be reverted in the future.

Reply:
Both renaming and separate commit done. I updated my solution to only affect 'portal_rating', but the solution is still not super great so it might still want to be reverted in the future.